templates/backup-miniflux.sh.j2

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+{{ ansible_managed | comment }}
+
+# backup script for miniflux
+# https://miniflux.app/faq.html#backup
+# https://www.postgresql.org/docs/current/backup.html
+
+export PATH=/usr/sbin:/usr/bin:/sbin:/bin
+umask 077
+
+date=`date +%Y%m%d`
+dest={{ backupdir }}
+[ ! -d "$dest" ] && install -d -m 0700 -o root "$dest"
+destfile=$dest/backup-miniflux-$date.tar.gz
+
+{% if miniflux_dbname is defined and miniflux_dbname != '' %}
+sudo -u postgres pg_dump -c "{{ miniflux_dbname }}" > $dest/pg_dump-miniflux.sql
+{% endif %}
+
+tar czf $destfile $dest/pg_dump-miniflux.sql
```

Comment:
Don't we need to put the tar command inside the `if` statement too?
Or we need to require the `miniflux_dbname` variable, the whole playbook is pointless without a dbname

Reply:
that comes of a script where backup has more than db aka files to save too.
here, I removed those if.
miniflux_dbname is in defaults anyway https://github.com/Ajnasz/ansible-role-miniflux/blob/master/defaults/main.yml#L7